metadata-db-app/src/cmr/metadata_db/data/oracle/granule_table.clj

```diff
@@ -25,7 +25,8 @@
        ;; migration. We should change the column to NOT NULL once it is
        ;; fully populated.
        "granule_ur VARCHAR(250),
-        transaction_id INTEGER DEFAULT 0 NOT NULL"))
+        transaction_id INTEGER DEFAULT 0 NOT NULL,
+        created_at TIMESTAMP WITH TIME ZONE"))
```

Comment:
so this one should be TIMESTAMP WITH TIME ZONE DEFAULT NULL (it is used for creating new tables for new providers, so there isn't any data to transition).